src/main/webapp/index.html

```diff
@@ -2,33 +2,31 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Play User Data Beetle</title>
-    <script src="script.js"></script>
-    <link rel="stylesheet" href="style.css">
+    <title>Splash Page</title>
```

Comment:
This should remain Play User Data Beetle right?

Reply:
done